docs/measurements/procedures/rigid_waist_shift.md

```diff
@@ -12,10 +12,63 @@
       After global corrections are done and trimmed in the machine, one applies a rigid waist shift in a given IR and scans the colinearity knob for the value that minimizes the $|C^{-}|$.
       These settings, when taking away the rigid waist shift, will minimize local coupling and its impact at the IP.
 
+## Rigid Waist Shift & LSA Knobs
+
+Some information about the rigid waist shift can be found in [this gallery page][rws_gallery]{target=_blank} from Felix's package's documentation.
+Some details about the creation of these knobs and their addition in LSA can be found in this [LHC OMC logbook entry][logbook_entry]{target=_blank} and the ones that follow.
+
+### Knob Setting Convention
+
+Two different knobs are used for the waist shift that need to be trimmed in: 
+
+- The triplets knob, which generates the waist shift itself.
+- The independent quadrupoles knob (Q4 - Q10), which re-matches the optics.
+
+The unit setting of the triplet knob is an arbitrary definition, where a value of $\pm1$ corresponds to a $\pm 0.5$% change of the triplet powering, which leads to a waist shift of ~43-44cm in a given direction.
+The knob setting is defined as follows, with regards to Beam1 (reverse for Beam2):
```

Comment:
The waist shift (and probably also some other things here) are taken for the beta* 30 cm optics(?).

Reply:
Yes! I should add this.

Reply:
Done in [c585a39](https://github.com/pylhc/pylhc.github.io/pull/74/commits/c585a3980e084c56cd196f7a82c3676e3cae44d3)